pkgs/development/tools/protoc-gen-rust/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchCrate
+, lib
+, rustPlatform
+, protobuf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rust-protobuf";
+  version = "3.1.0";
+
+  src = fetchCrate {
+    inherit version;
+    pname = "protobuf-codegen";
+    sha256 = "sha256-DaydUuENqmN812BgQmpewRPhkq9lT6+g+VPuytLc25Y=";
+  };
+
+  cargoSha256 = "sha256-cJUo8qEsVhVHZBuRKDtspFmhmbL4YHFIy2rrx0EkgXg=";
+
+  cargoBuildFlags = ["--bin" "protoc-gen-rust"];
+
+  nativeBuildInputs = [ protobuf ];
+  doCheck = true;
+
+  meta = with lib; {
+    description = "Protobuf plugin for generating Rust code";
+    homepage = "https://github.com/stepancheg/rust-protobuf";
+    license = licenses.mit;
+    maintainers = with maintainers; [ lucperkins ];
+    mainProgram = "protoc-gen-rust";
```

Comment:
This is the package name by default, so this can be removed as long as the package name was fixed.

```suggestion
```